client/src/App.js

```diff
@@ -29,11 +32,20 @@ function App() {
   }
 
   return (
-    <main>
-      <h1>reMEMBER</h1>
-      <Home orderedMembers={orderedMembers} />
-      <Add submitFunction={addMember} />
-    </main>
+    <div>
+      <Header />
+      <main>
+        <Switch>
+          <Route exact path="/">
+            <Home orderedMembers={orderedMembers} />
+          </Route>
+          <Route path="/add">
+            <Add submitFunction={addMember} />
+          </Route>
+        </Switch>
+      </main>
+      <Navigation />
+    </div>
```

Comment:
```suggestion
    <>
      <Header />
      <main>
        <Switch>
          <Route exact path="/">
            <Home orderedMembers={orderedMembers} />
          </Route>
          <Route path="/add">
            <Add submitFunction={addMember} />
          </Route>
        </Switch>
      </main>
      <Navigation />
    </>
```

If the `<div>` is not really necessary you could maybe delete it completely or replace it with fragments.

See https://reactjs.org/docs/fragments.html

Reply:
this.

Reply:
You are right, I will change that :-).